ServerCore/Pages/Hints/AuthorIndex.cshtml

```diff
@@ -0,0 +1,95 @@
+﻿@page "/{eventId}/{eventRole}/Hints/AuthorIndex/{puzzleId?}"
+@model ServerCore.Pages.Hints.AuthorIndexModel
+@{
+    ViewData["Title"] = "AuthorIndex";
+    ViewData["AdminRoute"] = "../Hints/AuthorIndex";
+    ViewData["AuthorRoute"] = "../Hints/AuthorIndex";
+    // TODO: Needs to handle implicit teams - ViewData["PlayRoute"] = "../Submissions/Index";
+    ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
+
+    if (Model.Puzzle != null)
+    {
+        Layout = "../Puzzles/_puzzleManagementLayout.cshtml";
+    }
+    else if (Model.Team != null)
+    {
+        Layout = "../Teams/_teamLayout.cshtml";
+    }
+    // TODO: Do we need a combined layout, or is that rare enough as to be unnecessary? Maybe those bar layouts should be sections/components?
```

Comment:
Combined would be nice for overall module monitoring per #328

Reply:
That's not what "combined" means here. It's asking about a combined nav layout, to allow navigation to both other puzzle properties and other team properties if you happen to be looking at one team's hints on one puzzle. This page will show hints taken for all puzzles if you click the link at the top of the puzzle index or clear the filter.

Agree that an all-up feedback view would be useful to have, in addition to our all-up submissons and now all-up hints. But I'm not convinced we need one view for all three at once.